style.css

```diff
@@ -141,16 +139,11 @@ body {
   width: 21.46px;
   height: 167px;
   position: absolute;
-
-  /* left: 30%;
-  top: -20px; */
   left: 134px;
   top: 18px;
 }
```

Comment:
Kindly, I suggest you use more descriptive class names so that in the future you or a partner can identify them more easily.